app/styles/_mixins.scss

```diff
@@ -0,0 +1,34 @@
+@mixin vertical-align {
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+@mixin two-by-three($width) {
+  height: $width * 0.667;
```

Comment:
I don't imagine this would stretch anything, but should this be `max-height` here?